Description: Fix FTBFS on bionic. Compiler errors:
 In file included from Unified_cpp_widget_gtk2.cpp:2:
 In file included from /<<BUILDDIR>>/firefox-91.0~b9+build1/widget/gtk/nsClipboardWayland.cpp:12:
 /<<BUILDDIR>>/firefox-91.0~b9+build1/widget/gtk/nsClipboardWayland.h:53:30: error: unknown type name 'nsWaylandDragContext'
   void SetWaylandDragContext(nsWaylandDragContext* aDragContext);
                              ^
 /<<BUILDDIR>>/firefox-91.0~b9+build1/widget/gtk/nsClipboardWayland.h:54:3: error: unknown type name 'nsWaylandDragContext'
   nsWaylandDragContext* GetWaylandDragContext();
   ^
 /<<BUILDDIR>>/firefox-91.0~b9+build1/widget/gtk/nsClipboardWayland.h:62:10: error: use of undeclared identifier 'nsWaylandDragContext'
   RefPtr<nsWaylandDragContext> mDragContext;
          ^
Author: Rico Tzschichholz <ricotz@ubuntu.com>

--- a/widget/gtk/nsClipboardWayland.h
+++ b/widget/gtk/nsClipboardWayland.h
@@ -37,6 +37,8 @@
   nsTArray<GdkAtom> mTargetMIMETypes;
 };
 
+class nsWaylandDragContext;
+
 class WaylandDataOffer : public DataOffer {
  public:
   explicit WaylandDataOffer(wl_data_offer* aWaylandDataOffer);
